sklearn/feature_selection/tests/test_rfe.py

```diff
@@ -537,7 +537,7 @@ def test_rfecv_std_and_mean(global_random_seed):
 
     rfecv = RFECV(estimator=SVC(kernel="linear"))
     rfecv.fit(X, y)
-    n_split_keys = len(rfecv.cv_results_) - 2
+    n_split_keys = len(rfecv.cv_results_) - 3
```

Comment:
Please add an inline comment to explain where the "3" comes from.

Reply:
Instead I changed it to directly select keys starting with "split" which is easier to understand.